pyglet/math.py

```diff
@@ -86,6 +86,31 @@ def __neg__(self):
     def __round__(self, ndigits=None):
         return Vec2(*(round(v, ndigits) for v in self))
 
+    def __radd__(self, other):
+        if other == 0:
+            return self
+        else:
+            return self.__add__(other)
+
+    def heading(self):
+        return _math.atan2(self[1], self[0])
```

Comment:
This can probably be a `@property` and it needs some docstrings and return type hints.